client/web/src/components/SaveToolbar.tsx

```diff
@@ -51,7 +51,7 @@ export const SaveToolbar: React.FunctionComponent<
         <>
             {error && willShowError() && (
                 <div className={styles.error}>
```

Comment:
```suggestion
                <div className={styles.error} role="alert">
```

Let's do this, as we're hiding the alert icon from screen readers